.ruby-version

```diff
@@ -1 +1 @@
-2.7.2
\ No newline at end of file
+2.7.7
```

Comment:
As in the previous PR, it seems unnecessary now

Reply:
Removed